contrib/gitian-build.py

```diff
@@ -161,6 +167,9 @@ def main():
         os.environ['USE_DOCKER'] = '1'
     elif not args.kvm:
         os.environ['USE_LXC'] = '1'
+        if args.is_bionic:
+            os.environ['GITIAN_HOST_IP'] = '10.0.3.1'
```

Comment:
Any reason why this conflicts the instructions in the docs repo?

https://github.com/bitcoin-core/docs/blame/fb356d71c1cb24da7560e19fa655f1022f7b7543/gitian-building/gitian-building-setup-gitian-debian.md#L39

Reply:
The default setting in `/etc/default/lxc-net` is 10.0.3.1

Reply:
Or should make it as the same as IP in document?

Reply:
I would stick to the default, unless there was a technical problem with it. The less customization the better. This part of the manual can be removed if this is merged. Maybe make sure that if `GITIAN_HOST_IP` is already set, you don't override it?

Reply:
Agree with @Sjors

Reply:
OK, I'll keep it default.